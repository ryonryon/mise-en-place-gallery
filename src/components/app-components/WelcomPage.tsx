import styled from "@emotion/styled";
import * as React from "react";

import BasicCard from "../cards/BasicCard";

const StyledDiv = styled.div`
  width: 80%;
  margin: 48px auto 64px auto;
  text-align: center;

  p {
    line-height: 140%;
    font-size: 20px;
  }
`;

const WelcomPage = () => (
  <StyledDiv>
    <BasicCard>
      <h1>Mise en place</h1>
      <p>
        Mise en place is a French culinary phrase which means "putting in place" or "everything in its place". It refers
        to the setup required before cooking and is often used in professional kitchens to refer to organizing and
        arranging the ingredients. Relates this mindset to programming, preparing UI components before you implement an
        actual project is the most efficient and organized way to develop great work.
      </p>
    </BasicCard>
  </StyledDiv>
);

export default WelcomPage;
