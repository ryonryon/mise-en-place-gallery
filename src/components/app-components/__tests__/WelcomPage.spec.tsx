import * as React from "react";
import * as renderer from "react-test-renderer";

import WelcomPage from "../WelcomPage";

describe("WelcomPage test", () => {
  it("tests WelcomPage renders correctly ", () => {
    const welcomPage = renderer.create(<WelcomPage />).toJSON();
    expect(welcomPage).toMatchSnapshot();
  });
});
