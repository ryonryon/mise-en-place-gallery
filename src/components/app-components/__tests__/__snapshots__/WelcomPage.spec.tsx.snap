// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomPage test tests WelcomPage renders correctly  1`] = `
<div
  className="css-1slhr4m"
>
  <div
    className="css-1pcjs3q"
  >
    <h1>
      Mise en place
    </h1>
    <p>
      Mise en place is a French culinary phrase which means "putting in place" or "everything in its place". It refers to the setup required before cooking and is often used in professional kitchens to refer to organizing and arranging the ingredients. Relates this mindset to programming, preparing UI components before you implement an actual project is the most efficient and organized way to develop great work.
    </p>
  </div>
</div>
`;
